packages/hint/docs/contributor-guide/getting-started/release.md

```diff
@@ -0,0 +1,229 @@
+# 🚀 Release process automation and improvements
+
+**STATUS: WORK IN PROGRESS**
+
+## Background
+
+[hint][hint] is a monorepo. This means it contains several packages under the same source control so it is easier to update them. In this case all those packages are under the `packages` folder: main CLI, `hint`s, `connector`s, `formatter`s, `configuration`s, etc. Everything maintained by the team lives there.
+
+Releasing a monorepo following `semver` can be a challenge. How to bump versions? How to deal with circular dependencies?
+
+This document aims to detail what webhint's current release process does, what are the pain points, and a proposal on how to aleviate them.
+
+## Bump versioning using commit tags
+
+Commit messages are used to decide how to bump the package's version:
+
+* `Docs:` no version change, related to `.md` changes
+* `Build:` no version change, related to build and release scripts
+* `Chore:` refactoring, tests, etc. anything not user-facing will be a `patch` (x.x.X)
+* `Upgrade:` dependency updates with no public API changes are a `patch` (x.x.X)
+* `Fix:` bug fixes are a `patch` (x.x.X)
+* `New:` added functionality that does not break previous behavior is a `minor` (x.X.x)
+* `Breaking:` any change to the public API or output (like error messages) is a `major` (X.x.x)
+
+## Automatic bumb versioning of local dependencies
+
+The release process updates local dependencies directly using `Upgrade` in the commit message. The reason is that [no code changes are needed to upgrade thus a `patch`](https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api). If any code modification is required it should be done prior running the release script.
+
+**⁉️ NOTE**: Publishing a new version shouldn't be done if the only change since it was las published is an `Upgrade` of dependency(ies).
```

Comment:
I think `las` shouldn't be there.

Reply:
I think it was meant to be `last`.

Reply:
```suggestion
**⁉️ NOTE**: Publishing a new version shouldn't be done if the only change since it was last published is an `Upgrade` of dependency(ies).
```